plugins/rbac-backend/src/service/policy-builder.ts

```diff
@@ -39,30 +38,35 @@ export class PolicyBuilder {
     },
     pluginIdProvider: PluginIdProvider = { getPluginIds: () => [] },
   ): Promise<Router> {
-    let adapter;
     const databaseEnabled = env.config.getOptionalBoolean(
       'permission.rbac.database.enabled',
     );
 
-    const databaseManager = await DatabaseManager.fromConfig(
-      env.config,
-    ).forPlugin('permission');
-    // Database adapter work
+    let dbConfig: Config;
     if (databaseEnabled) {
-      adapter = await new CasbinDBAdapterFactory(
-        env.config,
-        databaseManager,
-      ).createAdapter();
+      dbConfig = env.config;
     } else {
-      adapter = new FileAdapter(
-        resolvePackagePath(
-          '@janus-idp/backstage-plugin-rbac-backend',
-          './model/rbac-policy.csv',
-        ),
-      );
-      env.logger.info('rbac backend plugin uses only file storage');
+      // use in memory sqlite database, data won't be persistent
+      dbConfig = new ConfigReader({
+        backend: {
+          database: {
+            client: 'better-sqlite3',
+            connection: ':memory:',
+          },
+        },
+      });
+
+      env.logger.warn(`rbac backend plugin data won't be persistent`);
     }
 
+    const databaseManager =
+      await DatabaseManager.fromConfig(dbConfig).forPlugin('permission');
```

Comment:
We don't need the `await` here
```suggestion
    const databaseManager =
      DatabaseManager.fromConfig(dbConfig).forPlugin('permission');
```